### PR TITLE
fix(FeedbackScreen): avoid double responsive margins in buttons when they are not fixed

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1843)

### CODE REVIEW

File: src/feedback.tsx
Comment (marcoskolodny) [inline src/feedback.tsx:1]: Better review with [hidden whitespaces](https://github.com/Telefonica/mistica-web/pull/1195/files?diff=split&w=1)

**Review by yceballost — APPROVED** [review]
It is a production bug or just a super corner case?

**Comment by marcoskolodny** [comment]
> It is a production bug or just a super corner case?

@yceballost it happens when you use a feedbackScreen and the height of the viewport is not big enough, so the button is not positioned as fixed:

image

It's a corner case, but it's good to fix it

**Review by atabel — APPROVED** [review]

**Comment by atabel** [comment]
please, run a webapp preview build before merging this

**Pull request merged**